source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,23 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
```

Comment:
If they were `virtual`, you would have overrides. Since they are `abstract` and thus not implemented in the base class, they are not.